O365DataRetriever.ps1

```diff
@@ -732,20 +753,45 @@ $ConnectButton.Add_Click( {
 		Write-Host "Connecting to Exchange Online..." -ForegroundColor Cyan
 
 		#If user is MFA enabled
-		$UserAuthN = Get-MsolUser -UserPrincipalName ($creds).UserName 
-		if(($UserAuthN.StrongAuthenticationMethods) -ne $null){
-			Write-Host "You are MFA enabled..." -ForegroundColor Magenta
-			Write-Host "Unfortunately, we don't support users enabled for MFA at this point in time. Try again with a user account not MFA enabled." -ForegroundColor White -BackgroundColor DarkRed
-			$Window.Close()
-			break
+        $UserAuthN = Get-MsolUser -UserPrincipalName ($creds).UserName
+        $userIsMFA = $null -ne $UserAuthN.StrongAuthenticationMethods
+        if( $userIsMFA )
+        {
+            ## from https://docs.microsoft.com/en-us/powershell/exchange/exchange-online/connect-to-exchange-online-powershell/mfa-connect-to-exchange-online-powershell?view=exchange-ps
+            Write-Host "You are MFA enabled..." -ForegroundColor Magenta
+            Import-Module 'Microsoft.Exchange.Management.ExoPowershellModule' -ErrorAction SilentlyContinue
```

Comment:
Asking me to install a module, but don't know which one, until I say _**no**_ for the installation:

![mfa_issue](https://user-images.githubusercontent.com/25181757/44794108-ad461080-ab9f-11e8-810c-ccd043e63d6a.PNG)

Reply:
Code tries to automatically install EXO Powershell module from [powershellgallery](https://www.powershellgallery.com/packages/Microsoft.Exchange.Management.ExoPowershellModule/16.0.0.0), I guess to follow your design decision we have to check if the module exists and ask the user to install if the user is MFA enabled.

Reply:
Unfortunate, but true. While I trust the PSGallery on my computer, as @VeronicaGeek 's experience shows us - it's not trusted by default. :-(

Reply:
Fixed in latest pull.